radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ option(SHUTDOWN_CONFIRMATION "Shutdown confirmation" OFF)
 option(LCD_DUAL_BUFFER "Dual LCD Buffer" OFF)
 option(PXX1 "PXX1 protocol support" ON)
 option(PXX2 "PXX2 protocol support" OFF)
+option(AFHDS3 "AFHDS3 TX Module" ON)
```

Comment:
This needs to be discussed, do we want it supported as default? It uses a lot of RAM (am I right?). So it means more users not happy because some Lua scripts won't run anymore.

I am OK for Horus. We don't have this issue. But for Taranis I really want to be sure that this PR won't bring problems.

Reply:
Just for your reference size of ExternalModulePulsesData with AFHDS enabled for X9D - 864 bytes without 616 bytes. Lets disable it on F2.

Reply:
Implementation in Companion added, fwoptions.py filled with needed options.
Is there anything more to be done?